core/vm/interface.go

```diff
@@ -26,58 +26,88 @@ import (
 
 // StateDB is an EVM database for full state querying.
 type StateDB interface {
+	// CreateAccount creates a new account
 	CreateAccount(common.Address)
-
+	// SubBalance subtracts the amount from the address's balance
 	SubBalance(common.Address, *big.Int)
+	// AddBalance adds the amount to the address's balance
 	AddBalance(common.Address, *big.Int)
+	// GetBalance returns the balance for the given address
+	// For non-existent addresses or self-destructed addresses (during the tx execution)
+	// it returns zero
```

Comment:
The balance of an address which does not exist is zero. This is not surprising, since it has received no funds. I don't think that needs special mention in the docstring. 

For a self-desctructed account, I don't think the comment is true. Self-destruct sets the balance to zero. However, if the account receives more ether after the selfdestruct, this methods returns a non-zero balance. Which makes this comment erroneous. 

So IMO this sentence is a mix of superfluous info and erroneous